CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Changelog
 
+## 2.0.0 - [#24](https://github.com/openfisca/country-template/pull/24)
+
+#### breaking change
+
+* Details:
+  - Upgrade to Core v21
+  - Introduce the use of a string identifier to reference enum item.
+  - When setting an Enums (e.g. housing_occupancy_status), set the relevant string identifier (e.g. `free_lodger`). Indexes (e.g.`2`) and phrases (e.g. `Free Lodgers`) cannot be used anymore.
+  - The default value is indicated for each enum variable instead of being implicitly the first item of the enum.
+
+#### Web API request/response
+
+Before:
+
+```
+"persons": {
+    "Bill": {}
+},
+"households": {
+    "_": {
+        "parent": ["Bill"]
+        "housing_occupancy_status": "Free Lodger"
+    }
+}
+```
+Now:
+
+```
+"persons": {
+    "Bill": {}
+},
+"households": {
+    "_": {
+        "parent": ["Bill"]
+        housing_occupancy_status: "free_lodger"
+    }
+}
+```
+
+#### YAML testing
+Before:
+
+```
+name: Household living in a 40 sq.meters accomodation while free_lodgers
+  period: 2017
+  input_variables:
+    accomodation_size:
+      2017-01: 40
+    housing_occupancy_status:
```

Comment:
Quotes around `housing_occupancy_status`?

Reply:
Not in the Yaml :)